slave/process-mailaliases-generic/bin/process-mailaliases_generic.sh

```diff
@@ -37,10 +45,10 @@ function process {
 	
 	### Looking for duplicits in possible places (except perun_generic_file)
 	### take all files from /etc/aliases.d/ except old generic_file and all files with extension .db
-	fail_if_duplicits_found "${ETC_ALIASES}" "${FROM_PERUN}" `find "${ETC_ALIASESD_DIR}" -type f -not -name ${GENERIC_FILE} -not -name "*.db" -print`
+	fail_if_duplicits_found "${ETC_ALIASES}" "${FROM_PERUN}" `find "${DST_FILE_DIR}" -type f -not -name "${DST_FILE_NAME}" -not -name "*.db" -print`
```

Comment:
We need to rework the check for duplicates as it should expect default files to exists and skip them. I'll describe it in detail on redmine.